driver/src/main/scala/api/commands/bson/instanceadministration.scala

```diff
@@ -131,10 +131,10 @@ object BSONCollStatsImplicits {
       doc.getAs[BSONNumberLike]("paddingFactor").map(_.toDouble),
       doc.getAs[BSONNumberLike]("systemFlags").map(_.toInt),
       doc.getAs[BSONNumberLike]("userFlags").map(_.toInt),
-      doc.getAs[BSONNumberLike]("totalIndexSize").map(_.toInt).get,
+      doc.getAs[Int]("totalIndexSize").getOrElse(-1),
```

Comment:
Strictly reading as `Int` is not recommanded with MongoDB commands, which are used to change the type of the number properties from one release to another